src/network/networkd-routing-policy-rule.c

```diff
@@ -1170,6 +1230,13 @@ int routing_policy_serialize_rules(Set *rules, FILE *f) {
                         space = true;
                 }
 
+                if (rule->uid_range.start != UID_INVALID && rule->uid_range.end != UID_INVALID) {
+                        fprintf(f, "%suidrange=%"PRIu32"-%"PRIu32,
```

Comment:
UID_FMT instead of "%" PRIu32 please!

Reply:
Hmm, maybe no, as `uid_range.start` or `.end` are `uint32_t`, not `uid_t`.

Reply:
I noticed that we have `assert_cc(sizeof(uid_t) == sizeof(uint32_t));` at several places. So, I've replaced `"%"PRIu32` with `UID_FMT`.